samples/bluetooth/ipsp/README.rst

```diff
@@ -27,14 +27,132 @@ following commands:
    $ modprobe bluetooth_6lowpan
    $ echo 1 > /sys/kernel/debug/bluetooth/6lowpan_enable
 
-Then to connect:
+If you connected your board to a UART console, you will see an output similar to
+(may vary slightly by application and Zephyr versions):
+
+.. code-block:: console
+
+   [bt] [WRN] set_static_addr: Using temporary static random address
+   [bt] [INF] show_dev_info: Identity: cb:af:14:57:d8:6e (random)
+   [bt] [INF] show_dev_info: HCI: version 5.0 (0x09) revision 0x0000, manufacturer 0xffff
+   [bt] [INF] show_dev_info: LMP: version 5.0 (0x09) subver 0xffff
+   [bt] [WRN] bt_pub_key_gen: ECC HCI commands not available
+   [ipsp] [INF] init_app: Run IPSP sample
+   [ipsp] [INF] listen: Starting to wait
+
+The output above provides a BLE addressed assigned to your board (for the
+current session, will change on the next run).
+
+Alternatively, you may scan for your board on the host (this usually should
+be run as root, and you will need to break it manually):
+
+.. code-block:: console
+
+   # hcitool lescan
```

Comment:
> there doesn't seems to be anything about ZEP1656

Right, in my reply to @tbursztyka I said that I'm going to post this patch, and then we can see where to fit that info. I'm not exactly sure myself what to do that. My approach was to make it "just work" with https://github.com/zephyrproject-rtos/zephyr/pull/1151, and if we can't do that, I'm stumped how to go over it due to that "tl;dr" effect. An 2nd opinion/suggestion would be helpful.